tests/unit/IdGeneratorTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+use PHPUnit\Framework\TestCase;
+use Instrumentation\Aws\Xray\AwsXrayIdGenerator;
+use OpenTelemetry\Sdk\Trace\SpanContext;
+
+class AwsXrayIdGeneratorTest extends TestCase
+{
+    /**
+     * @test
+     */
+    public function GeneratedTraceIdIsValid()
+    {
+        $idGenerator = new AwsXrayIdGenerator();
+        $traceId = $idGenerator->generateTraceId();
+
+        $this->assertEquals(1, preg_match(SpanContext::VALID_TRACE, $traceId));
```

Comment:
Should validate if its in X-Ray trace id format, instead of W3C format

Reply:
I think this is fine, actually.  The ID generator just generates a 32 hex digit string that could be used with any propagator.  The x-ray propagator is the component responsible for making a trace ID fit the x-ray format.

Reply:
That's correct. Must be too long to remember this. Thanks for pointing this out